### PR TITLE
sommelier => 20220411

### LINKED ISSUE #6961: startsommelier uses env feature not available in older coreutils
Coreutils 8.24 does not support the `-S` option, and this is the default coreutils `/usr/bin/env` found on many chromebooks.

This causes a startup error whenever `startsommelier` is run.

### PR DESCRIPTION
**Pull request by supechicken**
Fix skycocker/chromebrew#6961

### Changes
- Update sommelier to 20220411 (commit 6e2cc8)
- Remove `-S` argument from `env`

Tested on `x86_64`, X and Wayland work with acceleration.

### CODE REVIEW

File: sommelier-wrapper [hunk 1]
```diff
@@ -1,4 +1,5 @@
-#!/usr/bin/env -S bash -eu
+#!/usr/bin/env bash
```
Comment (uberhacker) [inline sommelier-wrapper:1]: Would it be better to use `#!/bin/bash`?  That way if someone installs the crew bash package, we don't introduce potential unexpected issues.

File: sommelierd [hunk 2]
```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env -S ruby --disable-all
+#!/usr/bin/env ruby
```
Comment (uberhacker) [inline sommelierd:1]: `#!/usr/local/bin/ruby`?

Comment (satmandu) [reply sommelierd:1]: What if CREW PREFIX is different?  😅

File: sommelier-wrapper (hunk 1 above)
Comment (satmandu) [reply sommelier-wrapper:1]: I don't think this is a showstopper...

File: sommelierd (hunk 2 above)
Comment (satmandu) [reply sommelierd:1]: I don't think this is a showstopper either...

File: sommelier-wrapper (hunk 1 above)
Comment (supechicken) [reply sommelier-wrapper:1]: Changes made in #3 :)

**Pull request merged**